### ISSUE TITLE
Local *.desktop files should have a higher priority than global ones

### ISSUE THREAD

**Issue by b52** [DaveDavenport/rofi#579]
## Version

```
% rofi -v
Version: 1.3.1-291-gc27921f-dirty (master)
```

## Configuration

[Output of `rofi --help`](https://gist.github.com/b52/ff9244a31d9829e127adedfc762d6184)

## Launch Command

```
$ rofi -show drun
```

## Steps to reproduce

1. Copy any `*.desktop` file from `/usr/share/applications/` to `~/.local/share/applications/` **maintaining the original file name**
2. Change something in the just copied file, e.g., the name and/or the launch command
3. Run `rofi -show drun` and see whether it picked up the changes

## What behaviour you see

* Basically, it just shows the original desktop entry located in `/usr/share/applications/` and discards the local one totally

## What behaviour you expect to see

* The local desktop file should have a higher priority than the global one, which should make rofi to list only the local desktop file

**Comment by DaveDavenport** [comment]
Hmm this is regression from #453.

Can you run:
```
G_MESSAGES_DEBUG=Dialogs.DRun ./rofi -show drun
```
and paste the output.

**Comment by DaveDavenport** [comment]
I tried to reproduce, it seems to work for me.

It might be the cache. Try deleting ~/.cache/rofi2.druncache.
It seems doing shift-delete on the entry does not remove it from the cache, will look at that.

**Comment by b52** [comment]
Deleting `~/.cache/rofi2.druncache` solved the issue.

**Comment by DaveDavenport** [comment]
Good, in the latest version you can also select the 'cached' entry  and hit ctrl-delete . then on next read it should be fine.

Should handle this more gracefully.
